### ISSUE TITLE
Cannot bulk-change interface mode to "Tagged (All)"

### ISSUE THREAD

**Issue by fknorn** [netbox-community/netbox#4104]
### Environment
* Python version:  3.6.9
* NetBox version: 2.7.3

### Steps to Reproduce
1. In the device view, (checkbox-) select two or more interface and click **Edit**
2. On the right, select **Mode: Tagged (All)**
3. **Apply**

### Expected Behavior
The interface mode gets changed to Tagged (All) on the selected interfaces.

### Observed Behavior
Error:

```
<class 'TypeError'>

Direct assignment to the forward side of a many-to-many set is prohibited. Use tagged_vlans.set() instead.
```

**Comment by hSaria** [comment]
I believe this was fixed in v2.7.4. Specifically, in the revised solution for https://github.com/netbox-community/netbox/issues/4030

**Comment by fknorn** [comment]
@hSaria - you are absolutely right, apologies for not having test with the very latest version. I can confirm that this works just fine under v2.7.4. Cheers!
